masterlist.yaml

```
@@ -2011,47 +2011,88 @@ plugins:
-        name: 'Enhanced Blood Textures SE'
+        name: 'Enhanced Blood Textures'
   - name: 'dD - Enhanced Blood Main.esp'
     after:
       - 'Audio Overhaul Skyrim.esp'
       - 'Immersive Sounds - Compendium.esp'
+    req:
+      - <<: *SPID
+        condition: 'file("dD - Enhanced Blood Main__DISTR.ini")'
     inc: [ 'dD - Enhanced Blood Main LITE.esp' ]
-    msg: [ *includesMBBF ]
+    msg:
+      - *includesMBBF
+      - <<: *patchIncluded
+        subs: [ 'Skyrim Immersive Creatures' ]
+        condition: 'active("Skyrim Immersive Creatures Special Edition.esp") and not active("EBT - IC PATCH.esp")'
+    tag:
+      - Keywords
+      - Sound
     clean:
-      - crc: 0x70E9C0E7
-        util: 'SSEEdit v4.0.3'
+      - crc: 0x3A40639C
+        util: 'SSEEdit v4.0.4b'
+      - crc: 0xE3B9201C
+        util: 'SSEEdit v4.0.4b'
   - name: 'dD - Enhanced Blood Main LITE.esp'
     group: *fixesGroup
     clean:
       - crc: 0x6CF176EC
-        util: 'SSEEdit v4.0.3'
+        util: 'SSEEdit v4.0.4b'
   - name: 'dD-Reduced Wound Size.esp'
     clean:
-      - crc: 0x35D96F84
-        util: 'SSEEdit v4.0.3'
+      - crc: 0xB37A1C3A
+        util: 'SSEEdit v4.0.4b'
   - name: 'dD-(Larger|Reduced) Splatter Size\.esp'
     msg:
       - <<: *useOnlyOneX
         condition: 'many("dD-(Larger|Reduced) Splatter Size\.esp")'
         subs: [ 'EBT Splatter Size Modifier' ]
   - name: 'dD-(Medium|Short) Script Range\.esp'
-    inc:
-      - 'EBT - skyBirds Patch.esp'
-      - 'EBT - skyBirds Patch (mid).esp'
-      - 'EBT - skyBirds Patch (short).esp'
     msg:
       - <<: *useOnlyOneX
         condition: 'many("dD-(Medium|Short) Script Range\.esp")'
         subs: [ 'EBT Script Range Modifier' ]
+  - name: 'dD-Larger Drips.esp'
+    clean:
+      - crc: 0x58BBB1B8
+        util: 'SSEEdit v4.0.4b'
+  - name: 'dD-Larger Splatter Size.esp'
+    clean:
+      - crc: 0x5467E339
+        util: 'SSEEdit v4.0.4b'
+  - name: 'dD-LITE Larger Splatter Size.esp'
+    clean:
+      - crc: 0xAB05803D
+        util: 'SSEEdit v4.0.4b'
+  - name: 'dD-LITE Reduced Splatter Size.esp'
+    clean:
+      - crc: 0xBB9A5221
+        util: 'SSEEdit v4.0.4b'
+  - name: 'dD-Medium Script Range.esp'
+    clean:
+      - crc: 0x2771EB16
+        util: 'SSEEdit v4.0.4b'
+  - name: 'dD-No Screen Blood.esp'
+    clean:
+      - crc: 0x91164D46
+        util: 'SSEEdit v4.0.4b'
+  - name: 'dD-Reduced Splatter Size.esp'
+    clean:
+      - crc: 0xA8D3AC9C
+        util: 'SSEEdit v4.0.4b'
+  - name: 'dD-Short Script Range.esp'
+    clean:
+      - crc: 0x5B53ECF8
+        util: 'SSEEdit v4.0.4b'
+  - name: 'EBT - IC PATCH.esp'
+    clean:
+      - crc: 0xDABBF53F
+        util: 'SSEEdit v4.0.4b'
   - name: 'EBT - skyBirds Patch( \((mid|short)\))?\.esp'
-    msg:
-      - <<: *useOnlyOneX
-        condition: 'many("EBT - skyBirds Patch( \((mid|short)\))?\.esp")'
-        subs: [ 'EBT SkyBirds Patch' ]
+    msg: [ *obsolete ]
```

Some users might read the message as implying that there are more up-to-date versions of the patches available, so we should clarify that we're talking about the main mod:

```suggestion
    msg:
      - <<: *obsolete
        subs: [ 'Enhanced Blood Textures v4.0+' ]
```